### ISSUE TITLE
Assign category to several operations at once

### ISSUE THREAD

**Issue by nicofrand** [bnjbvr/kresus#149]
It would be great if we could select several operations (maybe with a column with checkboxes) and assigning a category to all of them (the checkbox would then be unchecked to avoid any further error).

**Comment by ZeHiro** [comment]
I would extend this to "modify operation type" to a set of operations at once

**Comment by bnjbvr** [comment]
Good idea! That would be pretty handy in the following scenario: do a search, apply a category / type to all results.

**Comment by nicofrand** [comment]
That's exactly what I am currently doing, one by one :)…
By the way, is there any reason why the category dropdown is only shown on click ? It takes 2 click to assign a category instead of 1…

**Comment by bnjbvr** [comment]
Mostly because of efficiency (it takes less memory and it's faster to render to have only one dropdown with N options at a time, rather than M dropdowns with N options where M is the number of operations). Maybe we could add a hover effect to make it clearer or a keyboard shortcut to help assigning categories...
